### PR TITLE
feat: Pbi 25 view current team profile frontend

### PR DESCRIPTION
**Pull request by ferdifdi**
### Describe your changes
Implement view current team
image

### Backlog name
Pbi 25 view current team profile frontend

### Checklist
- [X] I have performed a self-review of my code
- [X] Code successfully run on local
- [X] Feature / changes tested on local
- [X] No failing unit test
- [ ] Passed UAT Test
- [ ] Sonarqube tested 
- [ ] Required any changes to environment variable

### CODE REVIEW

File: src/routes/createTeam/index.js [hunk 1]
```diff
@@ -22,7 +22,7 @@ const CreateTeam = () => {
     } = useForm({
         defaultValues: {
             name: "",
-            description: "",
+            description: null,
```
Comment (andrew-susanto) [inline src/routes/createTeam/index.js]: ini kenapa dirubah tuh fer? wkwkk

File: src/routes/view_current_team/index.js [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+			setCurrentTeam(response.data)
+            console.log(response)
+		})
+	},[])
+
+    
+    return (
+        <div style={{padding: "40px 20px"}}>
+            <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.2.1/dist/js/bootstrap.bundle.min.js" integrity="sha384-u1OknCvxWvY5kfmNBILK2hRnQC3Pr17a+RTT6rIHI7NnikvbZlHgTPOOmMi466C8" crossorigin="anonymous" />
+			<link href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.1/dist/css/bootstrap.min.css" rel="stylesheet" />
```
Comment (andrew-susanto) [inline src/routes/view_current_team/index.js:27]: kayaknya udah gaperlu bootstrap lagi (?)

File: src/routes/createTeam/index.js (hunk 1 above)
Comment (luthfifahlevi) [reply src/routes/createTeam/index.js]: karena gw titip kmrin wkwk dibalikin ke string kosong ya fer, pake if else di FE aja

File: src/routes/view_current_team/index.js [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+                            <Text mb={23} fontSize='32px' fontWeight='semibold' color='black'>
+                                Team Members List
+                            </Text> 
+
+                            <table style="width:100%">
+                                <tr>
+                                    <th>Members</th>
+                                    <th>Status</th>
+                                </tr>
+                                {currentTeam.teammember && currentTeam.teammember.map((val)=>(
```
Comment (luthfifahlevi) [inline src/routes/view_current_team/index.js]: discuss dgn @Exqrch ya, klo gk slh, lucky buat itu jg, jdi mungkin pilih salah satu dri kalian aja

Comment (luckysusanto) [reply src/routes/view_current_team/index.js]: @ferdifdi ini udah aku implementkan sih, sorry ya lama :(, tapi seharusnya bisa dilihat :).

**Review by HugoIr — APPROVED** [review]
nitip resolve conflict

File: src/routes/createTeam/index.js (hunk 1 above)
Comment (ferdifdi) [reply src/routes/createTeam/index.js]: dah diubah kembali

**Review by andrew-susanto — APPROVED** [review]

**Pull request merged**